providers/oidc.go

```diff
@@ -247,6 +257,11 @@ func (p *OIDCProvider) findClaimsFromIDToken(ctx context.Context, idToken *oidc.
 		claims.UserID = fmt.Sprint(userID)
 	}
 
+	groups, ok := claims.rawClaims[p.GroupsClaim].([]string)
```

Comment:
This an area where I'm weak with go... when you mix a map lookup + type assertion, what is that `ok` referring to?

Should the type assertion part be handled in the if statement?

Reply:
AFAIK the `ok` relates to if the type assertion succeded or not, could be wrong here however.

Reply:
I think that's right, it would be the type assertion, but you'd have an untyped nil so that would fail (playground example: https://play.golang.org/p/VIkAptJRLH4)

I think on the next line the double check is redundant, we can just make it `if ok`